src/meta/ActiveHostsMan.cpp

```diff
@@ -35,20 +36,22 @@ kvstore::ResultCode ActiveHostsMan::updateHostInfo(kvstore::KVStore* kv,
         baton.post();
     });
     baton.wait();
-    return ret;
+    return MetaCommon::to(ret);
```

Comment:
I thought, should we completely unified `kvstore::ResultCode` and `cpp2::ErrorCode` ?
For historical reasons, kvstore module cannot depend on storage module, but now that doesn't seem to be the case. so can we using the cpp2::ErrorCode in kvstore module direct?

Reply:
Strongly Agree,  👍
At the storage, these two errorcodes are too annoying.

Reply:
Yeah, this pr is handled like this first, and then I will use a new pr to handle it, otherwise this pr is too big. wdyt?